src/main/java/bio/terra/service/dataset/DataDeletionRequestValidator.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.service.dataset;
+
+import bio.terra.model.DataDeletionRequest;
+import bio.terra.model.DataDeletionTableModel;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+import javax.validation.constraints.NotNull;
+
+@Component
+public class DataDeletionRequestValidator implements Validator {
+
+    @Override
+    public boolean supports(Class<?> clazz) {
+        return true;
+    }
+
+    private void validateFileSpec(DataDeletionTableModel fileSpec, Errors errors) {
+        String tableName = fileSpec.getTableName();
+        if (tableName == null || tableName.equals("")) {
```

Comment:
You can do this check with `StringUtils.isEmpty()` or if you want to know if it contains spaces, `isBlank`.